examples/Exploring-different-models.ipynb

```diff
@@ -0,0 +1,802 @@
+{
```

Comment:
<p>nit: I don't think the dataset name should be styled as inline code by the backticks. Plain text and initial caps ought to be fine.</p><p> </p><p>Do you mind replacing "here" as the link with an indication of the destination? Maybe "To download the training and test datasets, visit <a href="https://tianchi.aliyun.com/dataset/dataDetail?dataId=408#1" target="_blank">Ali-CCP: Alibaba Click and Conversion Prediction</a> at tianchi.aliyun.com."</p><p> </p><p>Maybe "We'll define several popular deep learning-based model architectures, train, and evaluate them and show how Merlin Models simplifies and eases this common and iterative process."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/256/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>